index.js

```diff
@@ -1,9 +1,12 @@
-const nlp = require('compromise');
-const nlpPronounce = require('compromise-pronounce');
-nlp.extend(nlpPronounce);
+import nlp from 'compromise';
+import speechPlugin from 'compromise-speech'
+nlp.plugin(speechPlugin)
 
-function pronounce(text) {
-  return nlp(text).pronounce().map(o => o.pronounce).toString();
+function pronounce(pronounceword) {
```

Comment:
I think it's a good idea to have [CamelCase](https://en.wikipedia.org/wiki/Camel_case) for function arguments.

Example:
- `pronounceWord`

What do you think?